templates/Web/Projects/AddPackageJsonDefault/package.json

```diff
@@ -5,9 +5,6 @@
   "dependencies": {},
   "devDependencies": {},
   "scripts": {},
-  "eslintConfig": {
-    "extends": "react-app"
```

Comment:
Just a question, why are we removing this, was it originally just for React?

Reply:
yeah it was just for react.

Reply:
If this is just for react, wouldn't we need to conditionally add it in if react is chosen as the front-end framework? I do not see any other place where this has been added.

Reply:
Yeah it should have just been there for react. I for some reason thought it would be a good idea to have it in the top level. Noticed it was being added for angular as well haha. Will be making a PR in about 2.5 hours to add this to just react.